Cabal/tests/PackageTests/Tests.hs

```diff
@@ -244,5 +242,28 @@ tests config =
             (concat $ lines (resultOutput r))
 
     tc :: FilePath -> TestM a -> TestTree
-    tc name m
-        = testCase name (runTestM config name Nothing m)
+    tc = testCase config
+
+-- | Tests that require shared libraries.
+sharedLibTests :: SuiteConfig -> [TestTree]
+sharedLibTests config =
+  tail [ undefined
```

Comment:
OK, I see, it's to make rearranging test cases easier (you don't need to delete commas).